.asyncapi-tool

```diff
@@ -0,0 +1,9 @@
+title: Java Spring Cloud Stream Template
+filters:
+  language: 
+  - Java
```

Comment:
```suggestion
  - javascript
```

for all templates it is `javascript` as it is about the language used in the code (not language of the app it generates)